template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Views/IdentityService/LoggedOut.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    ViewData["Title"] = "Sign Out";
```

Comment:
There isn't a great solution here. B2C uses sign-in and sign-out for the verbs, and we need to align with that. At the same time, ASP.NET Core Identity uses login extensively for the noun, which we can't really change. The best proposal I can come up with here is to consistently use sign-in and sign-up for the verbs and login and logout for the nouns. The one change would be to rename the AccountController.Login and Logout actions to SignIn and SignOut along with the corresponding view models. Thoughts?